### PR TITLE
Update Task_Controller.php

### PR DESCRIPTION
**Pull request by mhimon**
Fixed: self:: instead of $this-> for static methods.

### CODE REVIEW

File: src/Task/Controllers/Task_Controller.php [hunk 1]
```diff
@@ -541,7 +541,7 @@ public static function delete_task( $data ) {
             ->first();
 
         $resource = new Item( $task, new Task_Transformer );
-        $resource = $this->get_response( $resource );
+        $resource =  self::get_response( $resource );
```
Comment (ManiruzzamanAkash) [inline src/Task/Controllers/Task_Controller.php:544]: You could write it this way, as `$self` is already declared in this method and also `get_response()` is not a static method.

```php
$self->get_response( $resource );
```
Please remove also extra space before starting `$self` according to `phpcs`

File: src/Task/Controllers/Task_Controller.php [hunk 2]
```diff
@@ -583,7 +583,7 @@ public static function delete_task( $data ) {
             'list'     => $list
         ];
 
-        return $this->get_response( $resource, $message ); 
+        return  self::get_response( $resource, $message );
```
Comment (ManiruzzamanAkash) [inline src/Task/Controllers/Task_Controller.php:586]: also fix the above in here.

**Comment by mhimon** [comment]
Updated!

**Comment by ManiruzzamanAkash** [comment]
> Updated!

```php
return  $self->
```

There is two space before starting `$self`. Please give one space here. same for top.

**Comment by mhimon** [comment]
Bhai updated again! but maybe editor issue.

**Pull request merged**